test/e2e/windows/security_context.go

```diff
@@ -169,6 +169,7 @@ var _ = sigDescribe(feature.Windows, "SecurityContext", skipUnlessWindows(func()
 		// the pod would have failed to come up.
 		windowsPodWithSELinux := createTestPod(f, imageutils.GetE2EImage(imageutils.Agnhost), windowsOS)
 		windowsPodWithSELinux.Spec.Containers[0].Args = []string{"test-webserver-with-selinux"}
+		windowsPodWithSELinux.Spec.HostUsers = ptr.To(false)
 		windowsPodWithSELinux.Spec.SecurityContext = &v1.PodSecurityContext{}
```

Comment:
According to https://github.com/kubernetes/kubernetes/blob/ad72be434db58b7714712c84090b1a6f3d92145b/pkg/apis/core/validation/validation.go#L4158, `SecurityContext.HostUsers` should be set to the false value in this e2e test.